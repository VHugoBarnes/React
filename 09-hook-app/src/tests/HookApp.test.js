import { shallow } from 'enzyme';
import '@testing-library/jest-dom';
import React from 'react';
import { HookApp } from '../HookApp';

describe('Prueba en el componente HookApp', () => {
    
    test('Debe hacer match con el snapshot', () => {
        const wrapper = shallow(<HookApp />);
        expect(wrapper).toMatchSnapshot();
    });
    
});
