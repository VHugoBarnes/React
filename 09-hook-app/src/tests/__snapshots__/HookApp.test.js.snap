// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba en el componente HookApp Debe hacer match con el snapshot 1`] = `
<div>
  <h1>
    Hola Mundo
  </h1>
</div>
`;
